### PR TITLE
Update and-or.css

### PR DESCRIPTION
**Pull request by Christopher-O**
Updated CSS to allow brdr-0 to exist on all resolutions when the design is stacked, and not just "always stacked".

### CODE REVIEW

**Comment by duboisp** [comment]
Pre-approved upon review and after confirming there is an example of this change.

**Comment by duboisp** [comment]
pre-approved upon testing

Relates to 
* #2219 
* #2220 
* #2221

**Comment by Christopher-O** [comment]
@duboisp there is an example of the update on index.html under the updated header of no border.

**Review by duboisp — APPROVED** [review]
Tested the CSS change which are demoed in PR 2220 and work as expected. All CSS is demoed as expected.

Todo:
* [ ] Merge all other PRs into a single one.

File: méli-mélo/2021-05-conjunction/edge-cases.html [hunk 1]
```diff
@@ -16,8 +16,8 @@ <h2 class="h5">Table of content</h2>
 		<li>Edge cases of And/Or decision points (Conjunctions)</li>
 	</ul>
 </nav>
-
-<h2>No border shall not work for responsive break point</h2>
+<p><span class="label label-info">Updated:</span> The design has been updated to allow brdr-0 on all resolutions and will be removed when they become stacked.</p>
+<s><h2>No border shall not work for responsive break point</h2>
```
Comment (duboisp) [inline méli-mélo/2021-05-conjunction/edge-cases.html:20]: The `<s>` element should be use in the phrasing context not as a flow content.

I suggest to instead put this content inside a details/summary with a title that inform the content are not long accurate. Or simply you can remove those outdated instruction.

What would you like to do?
* [ ] Replicate the `<s>` element for every phrasing context in the deprecated instruction block
* [ ] Move the content inside a details summary with a title saying it is deprecated instruction
* [ ] Remove the outdated instruction.

**Review by duboisp — CHANGES_REQUESTED** [review]
Fix the HTML markup in the edge-cases.html. The [`<s>` element](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-s-element) can only be use in phrasing context according to HTML 5 spec.

Other than that, the rest is good to be squash and merge

File: méli-mélo/2021-05-conjunction/edge-cases.html (hunk 1 above)
Comment (duboisp) [reply méli-mélo/2021-05-conjunction/edge-cases.html:20]: I replicated the `<s>` on every sub element to ensure it is used as phrasing content.

**Review by duboisp — APPROVED** [review]
Tested and reviewed locally

**Pull request merged**